LstToLua/LstToLua.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Nullable>enable</Nullable>
```

Comment:
Let's just do this everywhere. :-)  There is little enough code that we can do this now. (We should also just use the TargetFramwork from the Directory.Build.props)